guardian/manage-frontend
# Add Supporter Plus product

## What does this change?
This PR adds the new Supporter Plus product into manage. Depends on [this pr in members-data-api](https://github.com/guardian/members-data-api/pull/778)


## How to test

Create an account with a Supporter Plus subscription and sign in to manage to check that it appears correctly

## Images

<img width="807" alt="Screen Shot 2022-07-22 at 09 58 18" src="https://user-images.githubusercontent.com/181371/180460540-7a04d68e-a49d-4e4c-b867-121b1ea410d2.png">
